core/src/main/java/cucumber/runtime/arquillian/ArquillianCucumber.java

```diff
@@ -69,13 +76,13 @@
 public class ArquillianCucumber extends Arquillian {
     private static final Logger LOGGER = Logger.getLogger(ArquillianCucumber.class.getName());
 
-    private static final String RUN_CUCUMBER_MTD = "____Cucumber_Runner_Not_A_Test";
```

Comment:
yeah, but adding ______ as prefix of a public method doesn't really hide the public method; also the question is: hide it from whom? Also it could be specified in the name that it should not be run directly without the need of that prefix, but obfuscating the name does not seem to be the right solution